osd/idp_has_connectivity_problems.json

```diff
@@ -3,6 +3,6 @@
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
     "summary": "Action required: update identity provider configuration",
-    "description": "Please verify the status of your identity server and update the IdP '${IDP_PROVIDER}' configuration to restore the cluster's operation. See documentation: https://docs.openshift.com/dedicated/4/authentication/dedicated-understanding-authentication.html",
+    "description": "Please verify the status of your identity server and update the IdP '${IDP_PROVIDER}' configuration to restore the cluster's operation. See documentation: https://docs.openshift.com/dedicated/identity_providers/config-identity-providers.html",
```

Comment:
the message should mention what is happening, the context "Red Hat SRE have detected that ...., please verify ..."